hw1/src/main/java/com/example/hw1/MovieRatingController.java

```diff
@@ -0,0 +1,45 @@
+package com.example.hw1;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.validation.BindingResult;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+
+import javax.validation.Valid;
+
+@Controller
+public class MovieRatingController {
+
+    @Autowired
+    MovieRatingRepository movieRatingRepository;
+
+    @GetMapping("/movies/new")
+    public String createMovieRating(Model model) {
+        MovieRating movieRating = new MovieRating();
+        model.addAttribute("movieRating", movieRating);
+        return "movie_rating_form";
+    }
+
+    @PostMapping("/movies/new")
+    public String processMovieRatingForm(@Valid MovieRating movieRating,
+                                         BindingResult result,
+                                         Model model) {
+        if (result.hasErrors()) {
+            return "movie_rating_form";
+        }
+        movieRatingRepository.save(movieRating);
+        Iterable<MovieRating> movieRatings = movieRatingRepository
+                .findAll();
```

Comment:
Your function for the query has a different name than what you used in your "hw1/src/main/java/com/example/hw1/MovieRatingRepository.java" file

make sure you are using the same function name

Reply:
Good catch I completely overlooked this, fixed it